tests/schema/tests.py

```diff
@@ -1024,6 +1024,8 @@ def test_alter_implicit_id_to_explicit(self):
         Author.objects.create(name='Foo')
         Author.objects.create(name='Bar')
 
+    @unittest.skipIf(connection.vendor == 'oracle',
+                     "Modifying to the identity column isn't supported in Oracle")
```

Comment:
I don't think this used to be the case -- if it is, then:
A) The error should be caught and reported properly
B) A test should be added for proper error handling
C) It needs to be documented as a backwards-incompatible change -- it may break existing sets of migrations